components/src/core/components/Table/ClassicTable.vue

```diff
@@ -49,7 +49,20 @@
           :key="header"
           :style="{width: header.width}"
         >
-          {{ item[header.name] }}
+
+        <div v-if="header.isHtml" v-html="item[header.name]"></div>
+        <div v-else>
+          <div
+            v-if="item[header.name].length > header.maxCharacterLength"
+            :tooltip="item[header.name]"
+            :flow="header.flow?header.flow:'bottom'"
```

Comment:
We can also use 
`header.flow || 'bottom'`
instead the ternary to make it much simpler

Reply:
Changed as suggested